bay/plugins/tail.py

```diff
@@ -22,17 +22,18 @@ def load(self):
 @click.option("--host", "-h", type=HostType(), default="default")
 @click.option('--follow/--no-follow', '-f', default=False, help="Follow log output")
 @click.option("--all/--no-all", default=False, help="Show the whole log")
+@click.option("--lines", "-n", type=int, help="Show specified number of lines")
```

Comment:
About the integer type will the option `-n 10` work or it has to be `-n10`?
Would be great to work with both.

Reply:
@remy-eb all of the following work:
```
-n 10
-n10
--lines 10
--lines=10
```